base_classes/NXobject.nxdl.xml

```diff
@@ -30,5 +30,182 @@
+						of a resource, cts. 
+						
+						A handle is a unique identifier used to facilitate the identification and management of digital objects. It is composed of a prefix that 
+						indicates the naming authority and a suffix that specifies the resource's local name. 
+						
+						This refers specifically to an ID in the Handle system operated by the Corporation for National Research Initiatives (CNRI).
+						
+						Syntax: prefix/identifier
+						Example: 123456789/abc123
+					</doc>
+				</item>
+				<item value="IGSN">
+					<doc>
+						International Generic Sample Number.
+						
+						The IGSN is a unique identifier assigned to a specific sample or specimen in the context of scientific research.
 
+						Since 2021, IGSNs are issued by DataCite, meaning that there are now DataCite-issued DOIs for all IGSNs, 
+						including those historical IGSNs issued beforehand. Therefore, the syntax is the same as for DOIs.
+						
+						Syntax: 10.XXXX/XXXXXX
+						Example: 10.1107/S1600576714027575
+					</doc>
+				</item>
+				<item value="ISNI">
+					<doc>
+						ISNI is an ISO standard to uniquely identify individuals and organizations involved in creative work, including pseudonyms
+						and other public personas.
+						
+						An ISNI-ID is made up of 16 digits, the last character being a check character. The check character may be either a decimal digit
+						or the character “X”.
+						
+						A URL can be generated from the ISNI ID by combining it with the prefix https://isni.org/isni/, resulting in
+						https://isni.org/isni/{ISNI-ID}.
+						
+						Syntax: 16 base-10 digits stored without any spaces.
+						Example: 0000000121032683
+					</doc>
+				</item>
+				<item value="ISSN">
+					<doc>
+						International Standard Serial Number
+						
+						An ISSN is an 8-digit unique identifier used to distinguish a serial publication, whether in print or electronic form.
+						The last character (a digit or 'X') serves as a check character, making the ISSN uniquely defined by its first seven digits.
+						
+						Syntax: NNNN-NNNC, where N a decimal digit character (i.e., in the set {0,1,2,...,9}), and C is in {0,1,2,...,9,X}
+						Example: 1234-5678 or 1234-567X
+					</doc>
+				</item>
+				<item value="ISSN-L">
+					<doc>
+						Linking ISSN
+						
+						The linking The ISSN, or ISSN-L, is a specific ISSN that groups the different media of the same serial publication.
+						
+						Syntax: NNNN-NNNC, where N a decimal digit character (i.e., in the set {0,1,2,...,9}), and C is in {0,1,2,...,9,X}
+						Example: 1234-5678 or 1234-567X
+					</doc>
+				</item>
+				<item value="ORCID">
+					<doc>
+						Open Researcher and Contributor identifier.
+						
+						ORCID provides a free and persistent identifier that uniquely distinguishes authors and contributors in scientific research.
+						
+						Example: https://orcid.org/XXXX-XXXX-XXXX-XXXX
+						Syntax: https://orcid.org/0000-0002-1825-0097
```

Comment:
"Syntax" and "Example" labels seem to be the wrong way around.  The "Example" label is the general format, while the "Syntax" label is before a specific example.

Reply:
You are right. I changed the labels.